python/ray/dag/compiled_dag_node.py

```diff
@@ -1231,17 +1241,27 @@ def _execute_until(
 
         Args:
             execution_index: The execution index to execute until.
+            timeout: The maximum time in seconds to wait for the result.
+                None means using default timeout, 0 means immediate timeout
+                (immediate success or timeout without blocking), -1 means
+                infinite timeout (block indefinitely).
```

Comment:
Add a comment about the default value

Reply:
Can you elaborate what you mean?

Reply:
I think the default value is `ctx_execution_timeout`.

Reply:
sure.